installer/helm/chart/volcano/config/volcano-scheduler-ci.conf

```diff
@@ -11,6 +11,6 @@ tiers:
   - name: drf
     enablePreemptable: false
   - name: predicates
-  - name: proportion
```

Comment:
This is a ci configuration and actually we have test both `proportion` and `capacity` plugins features.
And we have replaced the `proportion` to `capacity` when test e2e to cover the two plugins' feature.